fedimint-api/src/db/mod.rs

```diff
@@ -709,9 +709,17 @@ impl DecodingError {
 }
 
 mod tests {
```

Comment:
`#[cfg(tests)]` missing (test? or tests?)

Reply:
It's not missing on the `mod`, maybe I should create a separate `mod` though. The problem is the existing test `fn`s aren't standalone, but just a template to test all other database implementatios.

Reply:
Clippy is happy after I put the relevant `use`s into the test `fn`.

Reply:
I'm not sure if I get it. Should we rename this module if it's not a module with tests? `dbtest` or `testutil`? All other `mod tests` are exclusively for tests.